BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/application/validation/eutoxrisk/configuration/EUToxRiskFileValidatorProperties.java

```diff
@@ -0,0 +1,19 @@
+package uk.ac.ebi.biostd.webapp.application.validation.eutoxrisk.configuration;
+
+import lombok.Getter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Getter
+@ConfigurationProperties("eutoxrisk-file-validator")
+public class EUToxRiskFileValidatorProperties {
+    private boolean enabled = true;
+    private String projectId = "EU-ToxRisk";
+    private String endpoint = "https://eutoxrisk-validator.cloud.douglasconnect.com/v1/validate";
+    private final ThreadPool threadPool = new ThreadPool();
+
+    @Getter
+    static class ThreadPool {
+        private int corePoolSize = 1;
```

Comment:
I know is not a "BIG" problem right now but  I would try to keep properties for values that are expected to be changed (otherwise we end will hundreds of configuration properties) so I will suggest to remove this two properties `corePoolSize` and `maxPoolSize` and make them constants in this class.

Reply:
easy, done